README.md

```diff
@@ -1,26 +1,64 @@
 # Howso Engine&trade; Recipes
 
 This repository contains a collection of common use-cases primarily in Jupyter
-Notebook form. This is to help demonstrate not only the capabilities of
-[Howso Engine](https://github.com/howsoai/howso-engine) but to also illustrate important concepts and show how these
-tools can be used.
-
-
-## Installation
-
-This repository is not a package to be installed in the usual sense, but rather
-placed onto the local filesystem where the various notebooks can be
-interactively run, modified and explored.
-
-    cd your_local_workspace
-    unzip howso-recipes-engine-[VERSION].zip
-    cd howso-recipes-engine-[VERSION]
-
+Notebook form. They help demonstrate not only the capabilities of
+[Howso Engine](https://github.com/howsoai/howso-engine) but also 
+important concepts and usage instructions.
+
+There are two main methods for setting up to run the recipes:
+- Use a prebuilt VS Code dev container (requires VS Code and Docker)
```

Comment:
OPINION: I think the manual install should be the first/default vs. the Docker container. Mainly because once a user begins to craft their own solutions, they'll already be equipped with the skills required.

Reply:
Per our discussion, I added bullets in the first section providing info on the situation/user that each method may appeal to.